charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -27,7 +27,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    # TODO: remove TEMPORARY OVERRIDE FOR CR&D, reinstate:     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1264.service.core-compute-preview.internal/
```

Comment:
These values should be restored prior to merger.

Reply:
Done in [95b83a4](https://github.com/hmcts/ccd-data-store-api/pull/2058/commits/95b83a40aac320394117e9162b5b69b3243f1502)